packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/useCopyView.ts

```diff
@@ -1,5 +1,95 @@
+import { useState } from 'react';
+
+import { isFunction } from '@aws-amplify/ui';
+
+import { copyHandler } from '../../../actions/handlers';
+import { useProcessTasks } from '../../../tasks';
+import { useGetActionInput } from '../../../providers/configuration';
+import { useStore } from '../../../providers/store';
+import { getActionViewDisabledButtons } from '../utils';
+import { getTaskCounts } from '../../../controls/getTaskCounts';
+import { getDestinationListFullPrefix } from '../utils/getDestinationPickerDataTable';
 import { CopyViewState } from './types';
 
-export function useCopyView(): CopyViewState {
-  throw new Error('useCopyView is not implemented');
-}
+export const useCopyView = ({
+  onExit: _onExit,
+}: {
+  onExit?: () => void;
+}): CopyViewState => {
+  const [
+    {
+      location,
+      locationItems: { fileDataItems },
+    },
+    dispatchStoreAction,
+  ] = useStore();
+  const { key, current } = location;
+
+  const getInput = useGetActionInput();
+
+  const [{ tasks }, handleProcess] = useProcessTasks(
+    copyHandler,
+    // @ts-ignore
```

Comment:
This should be safe to remove now. In the future prefer `@ts-expect-error` so the TS compiler will tell us when a type issue is no longer relevant